src/index.js

```diff
@@ -143,6 +143,13 @@ export default ({
       return true;
     }
 
+    const filetypeKeys = Object.keys(stats.opts.filetypes);
+
+    filetypeKeys.push('.css');
+    if (!findMatchedFiletype(filename, filetypeKeys)) {
+      return true;
+    }
+
     return false;
```

Comment:
I think that instead of 

```typescript
if (!findMatchedFiletype(filename, filetypeKeys)) {
  return true;
}

return false;
```

you could write:

```typescript
return !findMatchedFiletype(filename, filetypeKeys);
```

Reply:
Thanks! The source has been updated.